pkg/graveler/ref/merge_base_finder.go

```diff
@@ -20,27 +20,28 @@ const (
 
 // FindMergeBase finds the best common ancestor according to the definition in the git-merge-base documentation: https://git-scm.com/docs/git-merge-base
 // One common ancestor is better than another common ancestor if the latter is an ancestor of the former.
-func FindMergeBase(ctx context.Context, getter CommitGetter, repository *graveler.RepositoryRecord, leftID, rightID graveler.CommitID) (*graveler.Commit, error) {
+// Returns the merge base commit and its CommitID.
```

Comment:
Please use a godoc style ("FindMergeBase returns..."), and document the added parameter here as well.  Also, what happened to the previous godoc?